core/src/main/java/com/segment/analytics/kotlin/core/utilities/AnySerializer.kt

```diff
@@ -16,8 +15,9 @@ object AnySerializer: KSerializer<Any> {
         return "not-implemented";
     }
 
-    @OptIn(InternalSerializationApi::class)
-    override val descriptor: SerialDescriptor = buildSerialDescriptor("My Any Serializer", SerialKind.CONTEXTUAL)
+    @OptIn(InternalSerializationApi::class, ExperimentalSerializationApi::class)
+    override val descriptor: SerialDescriptor = ContextualSerializer(Any::class, null, emptyArray()).descriptor
```

Comment:
I think this change doesn't matter. just nice to have, instead of init it arbitrary